FindTriplet.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class FindTriplet 
+{
+    public static void TakingInput(int[] A ,int size){
+        if(size==0){
+            return ;
+        }
+        Scanner s = new Scanner(System.in);
+        for(int i=0;i<size;i++){
+            A[i]=s.nextInt();
+        }
+    }
+    // returns true if there is triplet with equal to zero
+    public static int find3Numbers(int A[], int arr_size) 
+    {
+        int l, r;
+        
+        
+ 
+        // Fix the first element as A[i]
+        for (int i = 0; i < arr_size - 2; i++) 
+        {
+            // Fix the second element as A[j]
+            for (int j = i + 1; j < arr_size - 1; j++) 
+            {
+                // Now look for the third number
+                for (int k = j + 1; k < arr_size; k++) 
+                {
+                    if (A[i] + A[j] + A[k] == 0) 
+                    {
+                        System.out.print("Triplet is " + A[i] + " ," + A[j] + " ," + A[k]);
+                        return 1;
```

Comment:
After making the changes to the return type that I called out above, this looks like a good brute force solution.  What's the complexity?  Do you think we can do better?